probe/openocd.cmake

```diff
@@ -23,14 +23,26 @@ else()
 
   # Selects the OpenOCD debug probe file
   if(${DEBUG_PROBE} STREQUAL "digilent_jtag_hs3")
-    set(OPENOCD_DEBUG_PROBE_FILE "interface/ftdi/digilent_jtag_hs3.cfg")
+    set(OPENOCD_DEBUG_PROBE_CMD "source [find interface/ftdi/digilent_jtag_hs3.cfg]")
     check_probe_transport("jtag" ${OPENOCD_TRANSPORT})
   elseif(${DEBUG_PROBE} STREQUAL "cmsis-dap")
-    set(OPENOCD_DEBUG_PROBE_FILE "interface/cmsis-dap.cfg")
+    set(OPENOCD_DEBUG_PROBE_CMD "source [find interface/cmsis-dap.cfg]")
     check_probe_transport("jtag;swd" ${OPENOCD_TRANSPORT})
   elseif(${DEBUG_PROBE} STREQUAL "jlink")
-    set(OPENOCD_DEBUG_PROBE_FILE "interface/jlink.cfg")
+    set(OPENOCD_DEBUG_PROBE_CMD "source [find interface/jlink.cfg]")
     check_probe_transport("jtag;swd" ${OPENOCD_TRANSPORT})
+  elseif(${DEBUG_PROBE} STREQUAL "xvc")
+    if(NOT XVC_HOST)
+      message(FATAL_ERROR "No xvc host selected, utilize the -DXVC_HOST=<host_name> to select it.")
+    elseif(NOT XVC_PORT)
+        message(FATAL_ERROR "No xvc port selected, utilize the -DXVC_PORT=<port_name> to select it.")
```

Comment:
"No xvc port selected, use -DXVC_port=<port> to select it"
Also, fix the indentation.